src/erc20.ts

```diff
@@ -1,11 +1,12 @@
-import IERC20 from "@openzeppelin/contracts/build/contracts/IERC20.json";
 import { ethers } from "ethers";
```

Comment:
Looks like we don't need this anymore. Does it get caught by the linter?

Reply:
The ethers is still used in the type definition. And the IERC20.json from openzeppelin is also still used in the typechain script.